env0/data_source_code_variables.go

```diff
@@ -0,0 +1,81 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceCodeVariables() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceCodeVariablesRead,
+
+		Schema: map[string]*schema.Schema{
+			"template_id": {
```

Comment:
I'm assuming VCS is configured in a template (or environment with an implicit template).
Hopefully this assumption is correct.